<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoLAN Movie Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/images/vlmc</normaloff>:/images/vlmc</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionProject_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Export">
     <property name="title">
      <string>&amp;Export</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionRender"/>
     <addaction name="actionShare_On_Youtube"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="menu_Export"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionLoad_Project"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionImport"/>
   <addaction name="separator"/>
   <addaction name="actionRender"/>
   <addaction name="actionShare_On_Youtube"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionProject_Preferences"/>
   <addaction name="actionFullscreen"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/images/configure</normaloff>:/images/configure</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionMedia_Library">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Media Library</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </action>
  <action name="actionTranscode">
   <property name="text">
    <string>Transcode</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="statusTip">
    <string>Open the wizard to create a new project</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionRender">
   <property name="text">
    <string>&amp;To a file</string>
   </property>
   <property name="toolTip">
    <string>Render your project as a file</string>
   </property>
   <property name="statusTip">
    <string>Render your project as a file</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>&amp;Load Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="statusTip">
    <string>Import new medias into the current project</string>
   </property>
  </action>
  <action name="actionBypass_effects_engine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enable effects engine</string>
   </property>
  </action>
  <action name="actionProject_Preferences">
   <property name="text">
    <string>Proj&amp;ect Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionProject_Wizard">
   <property name="text">
    <string>Project Wizard</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
  </action>
  <action name="actionCrash">
   <property name="text">
    <string>Crash</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/images/undo</normaloff>:/images/undo</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../../resources.qrc">
     <normaloff>:/images/redo</normaloff>:/images/redo</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionShare_On_Youtube">
   <property name="text">
    <string>Share on &amp;Youtube</string>
   </property>
   <property name="statusTip">
    <string>Share your project on youtube</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
